internal/statemachine/classic_states.go

```diff
@@ -1323,16 +1325,12 @@ func (stateMachine *StateMachine) makeQcow2Img() error {
 			"-c",
 			"-O",
 			"qcow2",
-			"-o",
-			"compat=0.10",
```

Comment:
Is the compatibility flag the only thing that makes the resulting image non-sparse? Since I remember you also mentioned the `-S` qemu-img flag, but it seems to require giving a size..?

Reply:
Correct. While investigating using the `-S` option I learned that `compat=1.1` (the default) should first be used to let qemu-img efficiently work with sparse files. Then I saw that tuning the `-S` may not be worth it at first (it defaults to 4k). So for now I am not using it. Testing it also a bit tricky because the resulting qcow2 image is compressed, so the "sparsiness" of the file is not visible anymore (I had the same issue while testing the tar output).